lib/structures/CommandInteraction.js

```diff
@@ -9,13 +9,14 @@ const Message = require("./Message");
 const Collection = require("../util/Collection");
 const Permission = require("./Permission");
 
-const {InteractionResponseTypes} = require("../Constants");
+const {InteractionResponseTypes,ApplicationCommandContextType} = require("../Constants");
```

Comment:
If the linter didn't error on the lack of space after the comma that's a problem.. But yes add a space here please